rs/execution_environment/tests/canister_logging.rs

```diff
@@ -149,7 +149,7 @@ fn test_fetch_canister_logs_via_execute_ingress() {
         result,
         Err(UserError::new(
             ErrorCode::CanisterRejectedMessage,
-            "fetch_canister_logs API is only accessible in non-replicated mode",
+            "Cannot enqueue management message. Method fetch_canister_logs is not allowed to be called via ingress messages.",
```

Comment:
Let's make it one sentence
`Cannot enqueue management message because fetch_canister_logs method is not allowed via ingress messages`

Reply:
So, there's a [couple more cases](https://sourcegraph.com/github.com/dfinity/ic@fd7fc6ebef66df55e1061850168da9da5834f092/-/blob/rs/replicated_state/src/replicated_state.rs?L391-L404) where we have the pattern "Cannot enqueue management message. Mplah mplah". Maybe it makes sense to make all them a single sentence? But then I'd do it in a separate PR.

Reply:
I see, makes sense.